tests/chart_tests/test_poddisruptionbudget.py

```diff
@@ -80,10 +80,10 @@ def test_pod_disruption_budgets_default(self, kube_version):
         )
         _, minor, _ = (int(x) for x in kube_version.split("."))
 
-        if minor < 21:
-            assert all(x["apiVersion"] == "policy/v1beta1" for x in docs)
-        else:
+        if minor < 25:
             assert all(x["apiVersion"] == "policy/v1" for x in docs)
+        else:
+            assert ValueError("policy/v1beta1 is not supported in k8s 1.25+")
```

Comment:
This error text doesn't seem to match what we are testing for. We are testing for PDB only in `minor < 25`, so the error text should say something about PDBs not being supported in `minor >= 25`. AFAIK this no longer has anything to do with policy/v1beta1, which looks like it was only for `minor < 21`.